package.json

```diff
@@ -24,7 +24,7 @@
     "dev": "lerna run dev --parallel",
     "postinstall": "lerna bootstrap",
     "format": "lerna run prettify --parallel",
-    "lint": "concurrently yarn:lint:*",
+    "lint": "concurrently F:lint:*",
```

Comment:
Why edit this?

Reply:
I have no idea how this happened. I will correct it.